src/main/java/uk/gov/hmcts/probate/controller/BulkScanningController.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.probate.controller;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.probate.exception.BusinessValidationException;
+import uk.gov.hmcts.probate.model.ccd.raw.request.CallbackRequest;
+import uk.gov.hmcts.probate.model.ccd.raw.response.CallbackResponse;
+import uk.gov.hmcts.probate.service.BusinessValidationMessageRetriever;
+
+import java.util.Locale;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_UTF8_VALUE;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@Slf4j
+@RequiredArgsConstructor
+@RequestMapping(value = "/error", consumes = APPLICATION_JSON_UTF8_VALUE, produces = APPLICATION_JSON_VALUE)
```

Comment:
this path and the path on line 29 should be switched around, the class base is not error anymore and is bulk scanning/scanned docs so the base should reflect that with the method of the error handling then using the path of `error`